sourcecode/hub/resources/css/theme.scss

```diff
@@ -13,4 +13,35 @@
     .navbar-bottom {
         background-color: $green-200;
     }
+
+    p, h1 {
+        font-family: Helvetica Neue;
+    }
+
+    .btn-custom {
```

Comment:
"btn-custom" is not an appropriate description. I believe Bootstrap's `btn-light` will be more appropriate here, so this can be removed.

Reply:
The scondary button `btn-secondary` is now styled correctly for this button